texastribune/donations
# Update logo colors

#### What's this PR do?

What titles says, but also a few little one-offs:
- Adds the navbar to this app's 404 page
- Little padding fix for mobile 404 page

#### Why are we doing this? How does it help us?

Request from Art team/brand consistency

#### How should this be manually tested?

There really isn't too much to check here. The logo appears in the following places if you just want to quickly glance to make sure it's looking as expected

tt.svg
- Account page

logo.html
- Navbar for main pages (+ 404 page now)

TT_logo.jpg
- Guessing this is on the blastform sign up page in the little circle above the form

receipt-logo.png
- On the pdf for the reciept. I checked this fwiw


#### How should this change be communicated to end users?

I'll tell Emily A.

#### Are there any smells or added technical debt to note?

No

#### What are the relevant tickets?

https://3.basecamp.com/3098728/buckets/736178/todos/2836675932

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:

See related PR:
https://github.com/texastribune/texastribune/pull/4087
